Cargo.toml

```
@@ -20,18 +20,18 @@ codecov = { repository = "jonhoo/flurry", branch = "master", service = "github"
 maintenance = { status = "experimental" }
 
 [dependencies]
-parking_lot = "0.10"
+parking_lot = "0.12"
 num_cpus = "1.12.0"
 rayon = {version = "1.3", optional = true}
 serde = {version = "1.0.105", optional = true}
 seize = "0.2.0"
```

```suggestion
seize = "0.2.1"
```

Just published a release that strengthens a couple atomic loads.